mathesar_ui/src/systems/table-view/header/header-cell/ColumnHeaderContextMenu.svelte

```diff
@@ -30,7 +32,11 @@
   );
 
   $: hasGrouping = $grouping.hasColumn(columnId);
-
+  $: linkFk = processedColumn.linkFk;
+  $: getTablePageUrl = $storeToGetTablePageUrl;
+  $: fk_table_link = linkFk
```

Comment:
Change `fk_table_link` (snake_case) to `fkTableLink` (camelCase), per our [code standards](https://github.com/centerofci/mathesar-wiki/blob/master/engineering/standards/frontend.md).